app/src/main/java/edu/ucandroid/weathernotice/service/WeatherService.kt

```diff
@@ -37,17 +37,17 @@ class WeatherService {
                     call: Call<ArrayList<Weather>>,
                     response: Response<ArrayList<Weather>>
             ) {
-                _weather.value = response.body()
+                weather.value = response.body()
             }
 
         })
-        return _weather
+        return weather
     }
 
-    fun fetchWeather(locationName: String) : MutableLiveData<ArrayList<Weather>> {
-        var _weather = MutableLiveData<ArrayList<Weather>>()
+    fun fetchWeatherByLocation(locationName: String) : MutableLiveData<ArrayList<Weather>> {
```

Comment:
I refactored the function name (highlight, right-click, Refactor...) to be more clear when referenced elsewhere

Reply:
you right, good change.